iree_tests/README.md

```diff
@@ -1,5 +1,39 @@
 # IREE Tests
 
+## Quickstart instructions
+
+```
+#!/bin/bash
+set -e
+
+cd ~/SHARK-TestSuite/
+
+# Create and activate a virtual environment
+python3 -m venv .venv
+source .venv/bin/activate
+
+# Install required packages
+pip install -r iree_tests/requirements.txt
+pip install ./common_tools/
```

Comment:
I'm considering folding `common_tools` back into `iree_tests` to this extra setup step isn't needed, FYI.